src/apps/something-similar/SomethingSimilarList.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, FC, useState } from "react";
+import {
+  useRecommendFromFaustQuery,
+  RecommendFromFaustQuery,
+  useSearchWithPaginationQuery,
+  SearchWithPaginationQuery
+} from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import { Work } from "../../core/utils/types/entities";
+import fetchMaterial, {
+  MaterialProps
+} from "../loan-list/materials/utils/material-fetch-hoc";
+import RecommendMaterial from "../recommender/RecommendMaterial";
+
+export interface LikeThisListProps {
+  id: string;
+}
+
+const LikeThisList: FC<LikeThisListProps & MaterialProps> = ({
+  id,
+  material
+}) => {
+  const t = useText();
+  const [recommendView, setRecommendView] = useState<boolean>(true);
+  const { data: somethingSimilarData } = useRecommendFromFaustQuery({
+    faust: id,
+    limit: 4
+  });
+
+  const { data: byAuthorData } = useSearchWithPaginationQuery({
+    limit: 4,
+    q: {
+      all: material?.firstAuthor
+    },
+    offset: 0
+  });
+
+  const [somethingSimilar, setSomethingSimilar] =
+    useState<RecommendFromFaustQuery | null>(null);
+
+  const [authorMaterials, setAuthorMaterials] =
+    useState<SearchWithPaginationQuery | null>(null);
+
+  useEffect(() => {
+    if (somethingSimilarData) {
+      setSomethingSimilar(somethingSimilarData);
+    }
+  }, [somethingSimilarData]);
+
+  useEffect(() => {
+    if (byAuthorData) {
+      setAuthorMaterials(byAuthorData);
+    }
+  }, [byAuthorData]);
+
+  return (
+    <>
+      <h2 className="recommender__title text-header-h1 recommender__title--left">
+        {t("somethingSimilarTitleText")}
+      </h2>
+      <div className="recommender__buttons">
+        <button
```

Comment:
Isn't it possible to use our global button component?

Reply:
It does not look like a button but a link :)